Userland/Games/2048/main.cpp

```diff
@@ -143,7 +143,6 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             break;
```

Comment:
Commit message: "Fix: Fixes" does not help anyone; the commit title is enough for this change.

Reply:
Ah, apologies. Meant to remove that commit by squashing it through rebase but I guess it wasn't completely removed. Will fix it later today.